Sources/Jaeger/Jaeger/Extensions/TimeInterval+Extensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  TimeInterval+Extensions.swift
+//  Jaeger
+//
+//  Created by Simon-Pierre Roy on 10/29/18.
+//
+
+import Foundation
+
+/// A type to express time intervals in microseconds.
+typealias MicroSeconds = Double
```

Comment:
Microseconds is a single word, so we shouldn't  TitleCase it, instead:
```suggestion
typealias Microseconds = Double
```